website/docs/02 - Connection.md

```diff
@@ -55,21 +55,34 @@ After a `Connection` instance has been created, the connection must be opened be
 ### Syntax
 
 ```ts
-open(): Promise<void>
+open(options?: OpenOptions): Promise<void>
 ```
 
+### Parameters
+
+| Parameter | Type     | Description                                                | Example |
+| --------- | -------- | ---------------------------------------------------------- | ------- |
+| `options` | `object` | [Options object](#options-object-properties-1) (see below) |         |
+
+#### Options Object Properties
+
+| Property    | Type     | Default                 | Description                                                                                |
+| ----------- | -------- | ----------------------- | ------------------------------------------------------------------------------------------ |
+| `requestId` | `string` | randomly generated UUID | A request/trace ID to be passed to MVIS as a request header with the key `X-MVIS-Trace-Id` |
+
 ### Example
 
 ```ts
 import { Connection } from 'mvom';
 
 const mvisUri = 'http://foo.bar.com';
 const account = 'demo';
-const options = { timeout: 30_000 };
+const connectOptions = { timeout: 30_000 };
+const openOptions = { requestId: 'trace' };
 
 const makeConnection = async (): Connection => {
   const connection = Connection.createConnection(mvisUri, account, options);
```

Comment:
`options` is now undefined in this example.

Reply:
Fixed now